src/jit/block.cpp

```diff
@@ -1393,15 +1393,16 @@ bool BasicBlock::hasEHFlowIn()
 }
 
 //------------------------------------------------------------------------
-// blockHasEHFlowOut: Determine if this block ends in exception flow.
+// blockhasEHBoundaryOut: Determine if this block ends in an EH boundary.
 //
 // Arguments:
 //    block   - the BasicBlock
 //
 // Return Value:
-//    True iff the block ends in exception flow; false otherwise
+//    True iff the block end in an exception boundary that requires that all lclVars
+//    be valid on the stack; false otherwise.
```

Comment:
`all lclVars be valid on the stack;` does it mean all live lclVars are stored on the stack?

Reply:
"Stored on the stack" seems to imply that they must be stored at this point. What's required is that the value be valid on the stack - it may also have a valid value in a register. Would "all lclVars have an up-to-date value on the stack" be clearer?

Reply:
> "all lclVars have an up-to-date value on the stack"

Yes, I like that.